panther_analysis_tool/rule.py

```diff
@@ -184,25 +252,32 @@ def __init__(self, config: Mapping):
                 analysisType: either RULE or SCHEDULED_RULE
                 id: Unique rule identifier
                 body: The rule body
-                (Optional) version: The version of the rule
+                versionId: The version of the rule
+                (Optional) path: The rule module path
                 (Optional) dedupPeriodMinutes: The period during which
-                    the events will be deduplicated
+                the events will be deduplicated
         """
         self.logger = get_logger()
 
         # Check for required string fields
-        for each_field in ["id", "body", "versionId"]:
+        for each_field in ["id", "versionId"]:
             if not (each_field in config) or not isinstance(config[each_field], str):
                 raise AssertionError('Field "%s" of type str is required field' % each_field)
 
+        if not (config.get("body") or config.get("path")):
```

Comment:
The rule code module can now be imported directly from the filesystem or from raw source code, which is the case in Panther.